src/App.tsx

```diff
@@ -1,58 +1,30 @@
-import { useStore, useUnit } from "effector-react";
-import reactLogo from "./assets/react.svg";
-import { $count, addCountEvent } from "./store/counter";
+import { useUnit } from "effector-react";
 
 import "./App.css";
-import "./store/init";
-import { Button, ConfigProvider, Layout, theme } from "antd";
+import { ConfigProvider, Layout, theme } from "antd";
 import { $theme, ThemeSwitcher } from "./features/theme-switcher";
 
 const { Content, Header } = Layout;
-type Theme = 'dark' | 'light';
+type Theme = "DARK" | "LIGHT";
 
 export const App = () => {
-  const count = useStore($count);
   const [themeValue] = useUnit([$theme]);
 
-  const handleClick = () => {
-    addCountEvent();
-  };
-
   return (
     <ConfigProvider
       theme={{
-        algorithm: themeValue === "dark" ? theme.darkAlgorithm : theme.defaultAlgorithm,
+        algorithm:
+          themeValue === "DARK" ? theme.darkAlgorithm : theme.defaultAlgorithm,
       }}
     >
       <Layout>
-        <Header
-          style={{ display: "flex", justifyContent: "flex-end" }}
-        >
+        <Header className="Header">
```

Comment:
Why do we need a cool one here? what does it give?

Reply:
I don't understand what cool here means.